alf/drivers/on_policy_driver.py

```diff
@@ -66,17 +66,18 @@ def __init__(self,
                 callable(time_step.Trajectory).
             metrics (list[TFStepMetric]): An optional list of metrics.
             training (bool): True for training, false for evaluating
-            greedy_predict (bool): use greedy action for evaluation (i.e.
-                training==False).
+            epsilon_greedy (float):  a floating value in [0,1], representing the
+                chance of action sampling instead of taking argmax. This can
+                help prevent a dead loop in some deterministic environment like
+                Breakout. Only used for training=False
             train_interval (int):
         """
         super(OnPolicyDriver, self).__init__(
             env=env,
             algorithm=algorithm,
             observers=observers,
             metrics=metrics,
-            mode="on_policy_training" if training else
-            ("greedy_predict" if greedy_predict else "predict"))
+            mode=self.ON_POLICY_TRAINING if training else self.PREDICT)
```

Comment:
`epsilon_greedy=epsilon_greedy`  for `__init__`

Reply:
fixed